ldelf/ta_elf_rel.c

```diff
@@ -659,6 +661,14 @@ static void e64_relocate(struct ta_elf *elf, unsigned int rel_sidx)
 			 * here in case such entries exist.
 			 */
 			break;
+		case R_RISCV_RELATIVE:
```

Comment:
It's nice that so much of the 64-bit ELF code can be reused. However, we should not try to relocate a Risc-V ELF with AArch64 relocations types and vice versa. Please do something like:
``` C
#ifdef ARM64
    case R_AARCH64_NONE:
...
    case R_AARCH64_TLSDESC:
...
#endif
#ifdef RV64
    case R_RISCV_NONE:
...
    case R_RISCV_JUMP_SLOT:
...
#endif